tests/model_forms/tests.py

```diff
@@ -1710,6 +1710,18 @@ class WriterForm(forms.Form):
         sql, params = queryset.query.sql_with_params()
         self.assertEqual(len(params), 1)
 
+    def test_clean_handles_querysets_25980(self):
+        """
+        #25980 - ModelMultipleChoiceFields should be able to deal with querysets
+        when cleaning. This happens e.g. when the field is disabled and has a
+        queryset as initial value.
+        """
+        person1 = Writer.objects.create(name="Person 1")
+        f = forms.ModelMultipleChoiceField(queryset=Writer.objects.all(), disabled=True, required=False)
+        f.clean([person1.pk])
+        f.clean(Writer.objects.none())
+        f.clean(Writer.objects.all()) # used to throw a ValidationError: ['Enter a list of values.']
```

Comment:
eh sorry, so that fails right now, with the current 1.9.x branch. i don't know when it got introduced, i can search for that if you want (just not right now). i guess this got exposed with the introduction of the disabled attribute, as with it the initial value is passed unprocessed into clean.